rebasehelper/tests/public_api/test_specfile.py

```diff
@@ -48,6 +49,18 @@ def test_update_changelog(self, spec_object):
         assert spec_object.update_changelog('test2') is None
         assert spec_object.update_changelog(changelog_entry='test') is None
 
+    def test_get_new_log_with_non_c_locale(self, spec_object):
+        # Changelogs should be identical no matter what locale
+        changelog = spec_object.get_new_log("test2")
+        for l in locale.locale_alias:
+            try:
+                locale.setlocale(locale.LC_TIME, l)
+                # Prevents us from trying out strange locale aliases that fail
+                locale.setlocale(locale.LC_TIME, locale.getlocale(locale.LC_TIME))
+            except:
```

Comment:
This is making pylint complain, you could use `except Exception:` instead.

Reply:
Changed in FIXUP to ```except locale.Error```